### PR TITLE
Send message ski

### PR DESCRIPTION
**Pull request by svajusk**

### CODE REVIEW

File: discuss/customizations/mail_external_confirm.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    :align: center
+
+Functionality
+=============
+
+This feature warns the user, that the message he/she is intended to send 
+contains any type of external recipients. It works in *Send Message*, 
+*Log Note* with @ or *Channels*. 
+
+This feature is active with *Leads*, *Opportunities* in CRM module and
```
Comment (SButko) [inline discuss/customizations/mail_external_confirm.rst]: Actually, feature will work on all objects (models) that have mail chatter implemented (depends on mail.thread module), so my suggestion would be to mention on what king of objects does this work and mention examples, e.g.:

'This feature will work on all objects that have mail chatter implemented, e.g. Leads & Opportunities, Partners, Sales, Invoices and etc.'

File: discuss/customizations/mail_external_confirm.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+.. image:: media/mail_external_confirm_enable.png
+    :align: center
+
+Functionality
+=============
+
+This feature warns the user, that the message he/she is intended to send 
+contains any type of external recipients. It works in *Send Message*, 
+*Log Note* with @ or *Channels*.
```
Comment (SButko) [inline discuss/customizations/mail_external_confirm.rst]:
```suggestion
*Log Note* with @ (mention contact) or # (mention channel). 
```

File: discuss/customizations/mail_external_confirm.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+Try sending message and you will get a pop-up to confirm sending 
+with listed all external recipients:
+
+.. image:: media/mail_external_confirm_popup.png
+    :align: center
+
+.. note::
+    Confirmation in pop-up by default will have *Cancel* selection.
+
+Channel with external recipients will be trigered, when in channel
```
Comment (SButko) [inline discuss/customizations/mail_external_confirm.rst]:
```suggestion
Channel with external recipients will be triggered, when in channel 
```

**Comment by svajusk** [comment]
Updated also PO.

File: locale/lt/LC_MESSAGES/discuss.po [hunk 4]
```diff
@@ -579,6 +570,105 @@ msgstr ""
...
+msgid ""
+"Adding extra confirmation step while sending emails to external "
+"recipients (partners, followers). This is included in the chatter and "
+"advanced mail message composition wizard. Confirmation popover will "
+"appear if email is going to be sent to any of external partners with a "
+"list of external recipients."
+msgstr ""
+"Pridedamas papildomas patvirtinimas siunčiant žinutes išoriniams gavėjams"
+" (partneriams, sekėjams). Funkcionalumas veikia tiek bendravimo skiltyje,"
+" tiek ir pilname žinutės siuntuimo lange. Siunčiant žinutę išoriniams "
```
Comment (SButko) [inline locale/lt/LC_MESSAGES/discuss.po]:
```suggestion
" tiek ir pilname žinutės siuntimo lange. Siunčiant žinutę išoriniams "
```

**Review by SButko — APPROVED** [review]

**Pull request merged**